exporter/signozclickhousemetrics/exporter.go

```diff
@@ -75,6 +83,10 @@ type sample struct {
 	flags       uint32
 }
 
+func (s *sample) String() string {
+	return "value: " + strconv.FormatFloat(s.value, 'f', -1, 64) + " " + "timestamp:" + strconv.FormatInt(s.unixMilli, 10)
+}
```

Comment:
We don't need to collect the size for metrics. It's a waste of resources. Please use 0 as size and just keep the count